apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java

```diff
@@ -147,11 +146,9 @@ public void transformSampleARecord() throws FHIRException, IOException {
    * Serializes the EOB and prints to the command line.
    *
    * @throws FHIRException if there is an issue with transforming the claim
-   * @throws IOException if there is an issue with reading the test file
    */
-  @Disabled
   @Test
-  public void serializeSampleARecord() throws FHIRException, IOException {
+  public void serializeSampleARecord() throws FHIRException {
```

Comment:
Don't we want this disabled? This will output a lot of noise

Reply:
I can disable all the `serializeSampleARecord`. I'll attach a String message to them as well.

Reply:
More importantly, this isn't a test (it's not asserting anything.) This is a glorified debug tool.

Reply:
Removed